float32 Steering
float32 Throttle
float32 Speed
int32 Mod1
int32 Mod2